### PR TITLE
Listpage now populates items from firebase

### PR DESCRIPTION
**Pull request by Bigshmow**
TL;DR - Listpage populates ingredients in list form

All of the items sent to firebase are now being added dynamically to the grocerylist page in bullet list format.
We need to think about pushing the clicked items from recipe page into a specific folder (they now push to root) so that I am not pulling the recipes that are added on click from the landing/results page.  Unless we go local storage for saving and pulling those items, in that case the list page is 2/3 complete as far as functionality.  Still working on the googleplaces api to get searched store names to populate as markers.  I'll be actively finishing this up tonight.  Cant work on it at the office anymore lol!
Thanks for reading my long diary.

### CODE REVIEW

**Review by ppgeyser — APPROVED** [review]
Looking good so far well done duder
